modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,72 +3,151 @@
 // * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
 
 :_content-type: PROCEDURE
-[id="ztp-adding-new-content-to-gitops-ztp_{context}"]
-= Adding new content to the GitOps ZTP pipeline
 
-The source CRs in the GitOps ZTP site generator container provide a set of critical features and node tuning settings for RAN Distributed Unit (DU) applications. These are applied to the clusters that you deploy with ZTP. To add or modify existing source CRs in the `ztp-site-generate` container, rebuild the `ztp-site-generate` container and make it available to the hub cluster, typically from the disconnected registry associated with the hub cluster. Any valid {product-title} CR can be added.
+[id="ztp-adding-new-content-to-gitops-ztp_{context}"]
+= Adding custom content to the {gitops-shortname} ZTP pipeline
 
 Perform the following procedure to add new content to the ZTP pipeline.
 
 .Procedure
 
-. Create a directory containing a Containerfile and the source CR YAML files that you want to include in the updated `ztp-site-generate` container, for example:
+. Create a subdirectory,  `source-crs`,  in the directory containing the `kustomization.yaml` file for the `PolicyGenTemplate` CR.
```

Comment:
Lastly, since this is the first mention of CR in this section, I'd write the term out here:

```suggestion
. Create a subdirectory,  `source-crs`,  in the directory containing the `kustomization.yaml` file for the `PolicyGenTemplate` custom resource (CR).
```